site/src/components/form.md

```diff
@@ -172,28 +172,11 @@ The form behavior component can accept certain HTML attributes.
 				minlength="10"
 				rows="4"
 				cols="30"
+				required
```

Comment:
I see the `textarea` was marked as required but when I'm trying to submit a form it doesn't highlight it with red border or anything

![image](https://user-images.githubusercontent.com/57374379/181844030-2dc3d73f-3997-4a5f-a62d-3e6786f7fef7.png)
 
And it passes the empty submission

![image](https://user-images.githubusercontent.com/57374379/181844182-d9b8a66e-6b65-45b7-a29e-b0e9a7ae650a.png)

Reply:
Oh, I didn't notice that initially but it looks like the textarea contains some empty spaces by default

Reply:
Made an update to remove the empty spaces